src/app/app.component.ts

```diff
@@ -10,6 +10,9 @@ import { AppState } from './app.service';
  * Top Level Component
  */
 @Component({
+  host:{
+    class:'app app-component flex-container in-column-direction flex-grow-1'
```

Comment:
Class is a reserved word in TypeScript. However I can't find anything in the Angular docs providing an example. I only found http://stackoverflow.com/questions/35996212/is-it-possible-to-add-a-dynamic-class-to-host-in-angular-2